src/main/java/io/strimzi/kafka/bridge/BridgeConsumerEndpoint.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 Red Hat Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.strimzi.kafka.bridge;
+
+/**
+ * Interface for Sink Endpoints contains consumer specific methods
+ */
+public interface BridgeConsumerEndpoint {
```

Comment:
Not sure we need this class and if `initConsumer`, `subscribe` can be reused in some way. Maybe we could need a new one for polling but maybe not on this interface.